pkg/utils/partitions/getpartitions.go

```diff
@@ -61,6 +64,59 @@ func GetAllPartitions() (v1.PartitionList, error) {
 	return parts, nil
 }
 
+// GetMountPointByLabel will try to get the mountpoint by using the label only
+// so we can identify mounts the have been mounted with /dev/disk/by-label stanzas
+func GetMountPointByLabel(label string) string {
```

Comment:
at this point, reimplementing what GHW does its just a few lines more and we can adapt to our own specifics...